mlflow/ml-package-versions.yml

```diff
@@ -378,3 +378,18 @@ pmdarima:
     requirements: ["prophet"]
     run: |
       pytest tests/pmdarima/test_pmdarima_model_export.py --large
+
+diviner:
+  package_info:
+    pip_release: "diviner"
+    install_dev: |
+      if [ ! -d "$CACHE_DIR" ] || [ -z $(find $CACHE_DIR -type f -name "diviner-*.whl") ]; then
+        pip wheel --no-deps --wheel-dir $CACHE_DIR git+https://github.com/databricks/diviner.git
```

Comment:
How long does it take for `pip git+https://github.com/databricks/diviner.git` to complete?

Reply:
If it's short, just running `pip install git+https://github.com/databricks/diviner.git` is ok (and simpler).

Reply:
It has to build build (not install directly) prophet, statsmodels, and pmdarima. On my local machine it's a few minutes...

Reply:
Let's run this step twice and see how much this speeds up build.

Reply:
The current code only caches for a `diviner` wheel, not other package wheels.

Reply:
ahhhhhh that makes sense! Yeah I think the entire repo is probably < 10KB :) 
I'll do the direct import in a few minutes!

Reply:
updated!